modules/bvl_feedback/ajax/new_bvl_feedback.php

```diff
@@ -49,6 +57,10 @@
         $_POST['comment'],
         'Y'
     );
+    //build url for email
+    $link = $feedbackThread->getFeedbackURL();
+
+    $newNotifier->notify(array("feedback" => $link));
```

Comment:
this isn't working because `$_POST['commentID']` is an empty string. visit feedback goes to instrument level

Reply:
same issue applies for profile level feedback